pkgs/development/python-modules/jupyterthemes/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, matplotlib
+, jupyter-core
+, lesscpy
+, notebook
+}:
+
+
+buildPythonPackage rec {
+  pname = "jupyterthemes";
+  version = "0.20.0";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-Ko68DISyEquZufF1f8BYKj9Tkw06dbJJLZGnyLNqtB4=";
+  };
+
+  doCheck = true;
+
+  propagatedBuildInputs = [
+    matplotlib
+    jupyter-core
+    lesscpy
+    notebook
+  ];
+
```

```suggestion

  pythonImportsCheck = [ "jupyterthemes" ];

```

Another `pythonImportsCheck` here, if it makes sense